js/tests.js

```diff
@@ -1,5 +1,43 @@
 QUnit.test("hello test", function(assert) {
-  assert.strictEqual(1 + 1, 2, "One plus one is two");
+    assert.strictEqual(1 + 1, 2, "One plus one is two");
 });
 
 // ADD TESTS HERE
+QUnit.test("attack should decrease power level", function(assert) {
+    var testHero = new SuperHero("testHero", "black"); 
+    var testVillain = new SuperHero("testVillain", "gray"); 
+
+    testVillain.attack(testHero);
+
+    assert.strictEqual(testHero.powerLevel, 9, "hero.powerLevel is decreased by 1"); 
+
+    // remove the new test objects from DOM? 
+    $('#testHero').remove(); 
+    $('#testVillain').remove(); 
```

Comment:
Rather than needing to include cleanup like this in every test, you can write it once for a full set of tests:

```javascript
QUnit.testDone(function() {
  $('#SuperHeroWrapper').empty();
});
```

http://api.qunitjs.com/QUnit.testDone/

See also:

https://qunitjs.com/cookbook/#keeping-tests-atomic